emark/views.py

```diff
@@ -93,3 +97,67 @@ def get(self, request, *args, **kwargs):
                 "Cache-Control": "no-cache, no-store, must-revalidate",
             },
         )
+
+
+class DashboardView(LoginRequiredMixin, UserPassesTestMixin, TemplateView):
+    """Show a dashboard of available email classes."""
+
+    template_name = "emark/dashboard.html"
```

Comment:
You might want to use Django's admin base template. Our toolbox app, should be a good example.

Reply:
That would mean a dependency on the admin app.

Reply:
Cunning objection. I agree, that would be an unnecessary dependency.